apiserver/facades/client/bundle/bundle.go

```diff
@@ -701,6 +701,13 @@ func (b *BundleAPI) bundleDataApplications(
 		if result := b.constraints(application.Constraints()); len(result) != 0 {
 			newApplication.Constraints = strings.Join(result, " ")
 		}
+		if len(application.StorageConstraints()) != 0 {
+			newApplication.Storage = make(map[string]string)
+			for name, constr := range application.StorageConstraints() {
+				newApplication.Storage[name] = fmt.Sprintf("%v,%v,%v",
```

Comment:
It's more typical to use more specific format specifiers when you know the types, so presumably `%s,%d,%d` in this case (`go vet` should complain if you get them wrong).

Also, I'm a bit surprised there isn't a function to format these constraints already, but I couldn't find one at a glance either...